how-to/how-to-use-sysctls-with-kata.md

```diff
@@ -0,0 +1,128 @@
+# Setting Sysctls with Kata
+
+## Sysctls
+In Linux, the sysctl interface allows an administrator to modify kernel 
+parameters at runtime. Parameters are available via the /proc/sys/ virtual 
+process file system. 
+
+The parameters include the following subsystems among others:
+- kernel (kernel.)
+- networking (net.)
+- virtual memory (vm.)
+- file systems (fs.)
+
+To get a complete list of kernel parameters, you can run:
+```
+sudo sysctl -a
```

Comment:
Can you add a leading `$ ` to conform with https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#code-blocks.